README.md

```diff
@@ -1,127 +1,50 @@
 Floodlight OpenFlow Controller (OSS)
```

Comment:
nitpicking: Isn't the OSS redundant, considering it's publicly available on Github?

Reply:
It should be, but people have confused the two in the past, which is why I added it a while ago. I'd like to leave it, just to avoid future confusion

Reply:
👍